DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: hintr
 Title: R API for calling naomi district level HIV model
-Version: 0.0.4
+Version: 0.0.5
```

Comment:
Please increment the naomi requirement too (you require 0.0.3 now)

Reply:
Fixed, thanks.